### PR TITLE
[Mostly Nonmodular] Chameleon Mutation Buffs

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Makes the mutation a toggle, so that it isn't constantly on no matter what. Raises the instability to 35. If this is deemed too powerful, I'm willing to make it so that you're **constantly slightly transparent even if it's turned off**, so people know you have it. Like the old TK aura.

## How This Contributes To The Skyrat Roleplay Experience
I know a lot of people generally dislike genetics mutations, but I feel like some of them can be used, basically, as roleplay props. It's difficult to make use of Chameleon in such a way right now because it's, let's face it, difficult to roleplay with someone that is nonstop permanently invisible, and it can be a little straining on the immersion to have them be carrying like 9 syringes of it and 9 pills of mutadone just to toggle it on and off. I could see people using it to prank their friends like hiding in a room and waiting for them, or even using it with telepathy to narrate stuff to them (like their SECRET LOVE CRUSH ON THE CAPTAIN WOOOO.) 
also i want to make a qt chameleon girl.

## Changelog
:cl:
add:  Chameleon mutation can now be toggled on and off.
balance: Chameleon mutation instability raised to 35.
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
You got mixed indentation, you need to go to the bottom-right in VSC and check the "Tabs" button, and then click "Convert To Tabs" and it will convert all the indentation to tabs.

**Comment by RatFromTheJungle** [comment]
Does this also affect changelings? 👍  if it does

File: code/datums/mutations/chameleon.dm [hunk 1]
```diff
@@ -7,22 +7,56 @@
...
+	to_chat(user, text("The pigmentation of your skin shifts and starts to take on the colors of your surroundings."))
+
+/obj/effect/proc_holder/spell/self/chameleon_skin_activate/proc/chameleon_skin_deactivate(mob/user = usr)
+
+
+	if(!HAS_TRAIT_FROM(user,TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION))
+		return
+
+	REMOVE_TRAIT(user, TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION)
+		user.alpha = 255
```
Comment (tf-4) [inline code/datums/mutations/chameleon.dm]:
```suggestion
	REMOVE_TRAIT(user, TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION)
	user.alpha = 255
```

**Comment by Nerev4r** [comment]
gottem

**Comment by SpaceVampire** [comment]
I think this buff is fine, given the other stuff we have already removed from genetics.
I even think that not having it show if you have it will actually make it potentially useful for doing sneaky stuff.
It's not really like TK, where having no visual sign would actually be cancer.
Because the power is a lot more useful and offensively focused.

File: code/datums/mutations/chameleon.dm [hunk 2]
```diff
@@ -7,22 +7,56 @@
...
 
+/obj/effect/proc_holder/spell/self/chameleon_skin_activate/cast(list/targets, mob/user = usr)
+	. = ..()
+
+	if(HAS_TRAIT(user,TRAIT_CHAMELEON_SKIN))
+		chameleon_skin_deactivate(user)
+		return
+
+	ADD_TRAIT(user, TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION)
+	to_chat(user, text("The pigmentation of your skin shifts and starts to take on the colors of your surroundings."))
```
Comment (GoldenAlpharex) [inline code/datums/mutations/chameleon.dm]:
```suggestion
	to_chat(user, "The pigmentation of your skin shifts and starts to take on the colors of your surroundings.")
```

File: code/datums/mutations/chameleon.dm [hunk 3]
```diff
@@ -7,22 +7,56 @@
...
+
+/obj/effect/proc_holder/spell/self/chameleon_skin_activate/proc/chameleon_skin_deactivate(mob/user = usr)
+
+
```
Comment (GoldenAlpharex) [inline code/datums/mutations/chameleon.dm]: There's no reason to have two lines there, you're going to confuse people.
```suggestion
```

File: code/datums/mutations/chameleon.dm [hunk 4]
```diff
@@ -7,22 +7,56 @@
...
+	if(!HAS_TRAIT_FROM(user,TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION))
+		return
+
+	REMOVE_TRAIT(user, TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION)
+	user.alpha = 255
+	to_chat(user, text("Your skin shifts as it shimmers back into its' original colors."))
```
Comment (GoldenAlpharex) [inline code/datums/mutations/chameleon.dm]:
```suggestion
	to_chat(user, text("Your skin shifts as it shimmers back into its original colors."))
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Almost there!

**Comment by Nerev4r** [comment]
ready

**Pull request merged**